content/reference/drops/index.md

```diff
@@ -0,0 +1,42 @@
+# Drops
+
+*Drops* let you provide the user with custom functionality. From the user's perspective a drop acts very much like a Hash, though methods are accessed with dot-notation as well as element selection. A drop method cannot be invoked with arguments. Drops are called just-in-time, thus allowing you to easily load objects.
```

Comment:
This sounds more like a sentence for an actual web-designer, not bsa-website client.